Src/HostingStartup/HostingStartup.Net45.Tests/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage 
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given 
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1652:Enable XML documentation output", Justification = "Pre-existing condition")]
```

Comment:
Would it make more sense to put the GlobalSuppression.cs in either TestFramework.Shared or Common projects?